docs/_includes/bottom.html

```diff
@@ -0,0 +1,29 @@
+<!-- required scripts for Bootstrap -->
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+    crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+    crossorigin="anonymous"></script>
+
+<!-- add and remove classes to take advantage of Bootrap styling -->
+<!-- <script>
+    (function ($) {
+        $(document).ready(function () {
+            $("table").removeClass("parameters").addClass("table").addClass("table-striped").addClass("table-bordered");
+            $("span.tag").removeClass("tag").removeClass("extension").addClass("badge").addClass("bg-danger");
+        });
+    })(jQuery);
+</script> -->
```

Comment:
I guess this block can be removed. Maybe the scripts too, once this is removed?

Reply:
Yes, no custom scripts need to be used, this will be removed.